### PR TITLE
Fixed default variables and formatting on README.md

### PR DESCRIPTION
**Pull request by wurbanski**
Hi,

I have found some issues with the role, here are little fixes if you'd like to use them.

Thanks for your work!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,33 +12,33 @@ Requirements
...
+| jenkins_swarm_task_name              | Jenkins Swarm Client                                                                                 | Description for systemd                                                                           |
+| jenkins_swarm_systemd_path           | /lib/systemd/system                                                                                  | Path to systemd folder                                                                            |
+| jenkins_swarm_service_name           | swarm-client                                                                                         | Name of the systemd service                                                                       |
+| jenkins_swarm_client_wrapper_version | 2.0.3-pre                                                                                            | Windows Service Wrapper version                                                                   |
```
Comment (reynn) [inline README.md]: default was updated to 2.0.3 and the jenkins_swarm_wrapper_download_url var was changed.

File: README.md [hunk 2]
```diff
@@ -12,33 +12,33 @@ Requirements
...
+| jenkins_agent_labels_file            | {{ jenkins_swarm_path }}/labels.swarm                                                                | A file to hold the labels and add/remove dynamically (Swarm client 3.3 and above)                 |
+| jenkins_agent_log_file               | {{ jenkins_swarm_path }}/swarm.log                                                                   | Where the swarm agent will log to                                                                 |
+| jenkins_agent_additional_args        | [deleteExistingClients, disableClientsUniqueId]                                                      | Additional arguments to send to the Swarm client jar                                              |
+| jenkins_swarm_client_version         | 3.3                                                                                                  | Version of the Swarm client to download                                                           |
```
Comment (reynn) [inline README.md]: default was updated to 3.6

**Review by reynn — CHANGES_REQUESTED** [review]
Appreciated the PR, few minor things and I will be happy to merge it in.

**Comment by wurbanski** [comment]
Oh, I missed those :) It's fixed now.

**Review by reynn — APPROVED** [review]

**Comment by reynn** [comment]
Thanks @wurbanski! merged in

**Pull request merged**